### ISSUE TITLE
You can still pilot a mech in crit

### ISSUE THREAD

**Issue by ArcLumin** [HippieStationArchive/HippieStation13-1#2913]
Title. So long as you're not unconscious, you can continue to pilot your mech

**Comment by GuyonBroadway** [comment]
Heh, makes me think of some kinda pilot life support system... heck hows that for an upgrade? Something that injects a little tricord like a medbot whenever the Pilot is injured.

Heck I wonder if it's possible to give the mech a rudementry AI so it can follow it;s DNA locked pilot. and attack upon command like slimes...

**Comment by myontasm** [comment]
Kamina approved.

**Comment by LuisReyesX** [comment]
Titanfall when

**Comment by ArcLumin** [comment]
> Something that injects a little tricord like a medbot whenever the Pilot is injured.

You can load medibots in your mech and they'll automatically inject you, even if you're on the move

**Comment by LilKek450** [comment]
Yes but you need a clamp for that and one weapon slot is a lot tbh

**Comment by GuyonBroadway** [comment]
@IvanTheCracka

-Equip clamp
-load bot
-detatch clamp
-equip gun

Ezpz

**Comment by ArcLumin** [comment]
Only the ripley family can use clamps, and the one with the least slots is the firefighter, with 5. This is enough for:
Diamond drill
clamp
grav cata
repair droid
power regener
Which is enough to fuck up whatever. Ripleys can't use guns. Drills to the floor are instadeath/crit which ignores armor, harm clamps are fast, deal 20 damage and an extra 10 oxyloss, grav catapult lets you throw massive groups of people into your drill range.

Do NOT underestimate the clamp, it is insane. Unload soap and push it around with the catapult, then drill em, unload plasma cans and drill em; anyone who hits you in plasma will start a raging plasma fire. Set off voice actived smoke nades, load up allies in it and unload to bum rush, hide absorbed bodies, put facehuggers in it and unload em for an instant long stun, load up an emagged beepsky or ed209 to spam stuncuff, possibilities are endless.
